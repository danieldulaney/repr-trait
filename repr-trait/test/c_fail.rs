use repr_trait::C;

#[derive(C)]
struct NotC {
}

fn main() {}
