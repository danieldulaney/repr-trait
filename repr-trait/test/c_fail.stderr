error: proc-macro derive panicked
 --> $DIR/c_fail.rs:3:10
  |
3 | #[derive(C)]
  |          ^
  |
  = help: message: Can't derive C on a struct without #[repr(C)]
